### PR TITLE
Coordinate system/dda

### PR DESCRIPTION
**Pull request by fsandel**
- changed coordiante system from weird pixel based grid to 1x1 float cells depending on map size
- changed raycasting algorithm from simulating to dda calculating
- general cleanup

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -12,6 +12,8 @@ static t_window	*setup_window_struct(t_map *map)
 	window->player = malloc(sizeof(t_player) * 1);
 	window->player->pos = window->map->start_pos;
 	window->player->dir = window->map->start_dir;
+	map->width = 10;
+	map->height = 8;
```
Comment (philippschwarzmueller) [inline src/main.c:16]: please remove your mock-data :)

**Review by philippschwarzmueller — CHANGES_REQUESTED** [review]
remove the mock-up data pls other than that ty lgtm

File: src/main.c (hunk 1 above)
Comment (fsandel) [reply src/main.c]: removed the mock data

**Review by philippschwarzmueller — APPROVED** [review]
tytytytytytyty

**Pull request merged**